### PR TITLE
Fix cli/http upgrade

### LINKED ISSUE #4406: [CoreInstaller] Upgrade doesn't work by CLI
upgrading by CLI from <3.0.0 to 3.0.0 doesn't work

As pointed out by @kratorD, when following the instructions, the upgrade via CLI doesn't work. The error is

> In AbstractMySQLDriver.php line 93:
> [Doctrine\DBAL\Exception\ConnectionException]
> An exception occurred in driver: SQLSTATE[HY000] [1049] Unknown database 'o
> thing'

The basic problem is when following the instructions to 
> From the new installation, copy the `.env` file to `.env.local`

This doesn't include the existing DB information (defaults to `nothing`)

Previous travis testing on the upgrade reused the existing `.env.local` file which included the proper DATABASE_URL value so the error was not seen.
I've [adjusted the travis build](https://github.com/zikula/core/commit/415015c9114454d01ebbb11ab0e36a6a85748eb1) to demonstrate the error

result: https://travis-ci.org/github/zikula/core/jobs/708324659

Initial attempts at http upgrade process also fail

### PR DESCRIPTION
**Pull request by craigh**
| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | fixes #4406
| Refs tickets      | -
| License           | MIT
| Changelog updated | yes

## Description

accomodate 

## Todos

- [x] apply same style of fixes to http upgrade
- [x] remove code from upgrade command that is now not necessary
- [x] Tests
- [x] Documentation
- [x] Changelog

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -60,8 +60,8 @@ install:
     - mysql zk_test < tests/test_dbs/core143.sql
     # run the upgrade from the 143 db
     - cp .env .env.local # start with fresh copy of .env.local
-    - 'sed -i -E "s/ZIKULA_INSTALLED=''(.*)''/ZIKULA_INSTALLED=''1.4.3''/" .env.local'
     - cp tests/services_custom.yaml config/services_custom.yaml
```
Comment (Guite) [inline .travis.yml:63]: Is `- 'sed -i -E "s/core_installed_version:(.*)/core_installed_version: ''1.4.3''/" config/services_custom.yaml'` missing here?

**Comment by Guite** [comment]
Changelog entry :-)

**Comment by craigh** [comment]
travis seems to have forgotten this PR - maybe because I changed the name - but the build is still passing there. https://travis-ci.org/github/zikula/core

File: CHANGELOG-3.0.md [hunk 2]
```diff
@@ -6,13 +6,17 @@
   - _there should be none_
 
 - Fixes:
+  - [Admin] Fix error when extension is not categorized.
```
Comment (Guite) [inline CHANGELOG-3.0.md:9]: Please put extensions after the bundles (between `HookBundle` and `Blocks`).

File: src/Zikula/CoreInstallerBundle/Resources/views/login.html.twig [hunk 3]
```diff
@@ -4,7 +4,7 @@
 {% block content %}
 <div class="card" id="loginCredentials">
     <div class="card-header">
-        <strong>{% trans %}Please Login with Admin credentials{% endtrans %}</strong>
+        <strong>{% trans %}Please enter your Admin login credentials{% endtrans %}</strong>
```
Comment (Guite) [inline src/Zikula/CoreInstallerBundle/Resources/views/login.html.twig:7]: Translation update needed.

**Pull request merged**